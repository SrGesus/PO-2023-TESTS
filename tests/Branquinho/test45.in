2
n
saved
1
5
5
0
0