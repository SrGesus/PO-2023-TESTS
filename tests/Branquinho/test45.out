Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Guardar antes de fechar? Ficheiro a abrir: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 